frontend/src/services/query/QueryServiceImpl.ts

```diff
@@ -131,14 +137,15 @@ export default class QueryServiceImpl extends QueryService {
       (idOrDescriptor: number | { id: number }) =>
         typeof idOrDescriptor === 'number' ? idOrDescriptor : idOrDescriptor.id
     );
+
+    const kvps = ids.map((id) => ({ id, value: cache.get(id) }));
     // Already cached values.
     // Stored here since they might get removed from the cache during the async operation.
-    const foundValues = ids
-      .filter((id) => cache.has(id))
-      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion -- nullcheck done in line before
-      .map((id) => cache.get(id)!);
+    const foundValues = kvps.filter((p) => p.value != null).map((p) => p.value);
     // Values not in cache.
-    let missingIds: number[] = ids.filter((id) => !cache.has(id));
+    let missingIds: number[] = kvps
+      .filter((p) => p.value == null)
```

Comment:
Use `===`

Reply:
`== null` involves check on `null` and `undefined`, so if the other value is `null` or `undefined`, the equation is true. That what I want. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/null